### DISCUSSION TITLE
Exclude metrics from Datadog Integration

### DISCUSSION THREAD

**Discussion by takeshiemoto** [hasura/graphql-engine#9032] [Question] [answered]
I want to exclude metrics from Datadog Integration.

We are sending Hasura Cloud logs using Datadog Integration to Datadog.

Is there any way to send only logs and exclude metrics when using the integration?

I want to stop the AMP in Datadog because I am being charged for it.

**Comment by meetzaveri** [comment, marked as answer]
Hello @takeshiemoto 

In this case, we would encourage you to open a support ticket from hasura cloud dashboard. Thanks!
